sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Percentile.scala

```diff
@@ -88,7 +88,8 @@ case class Percentile(
     this(child, percentageExpression, frequency, 0, 0)
   }
 
-  override def prettyName: String = "percentile"
```

Comment:
We should override `nodeName` here. The default implementation of `prettyName` can handle `FunctionRegistry.FUNC_ALIAS` already